### PR TITLE
shadcn setup

### PR DESCRIPTION
**Pull request by meshackm**
Shadcn is ready for use

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -21,7 +21,20 @@
...
+    "linebreak-style": "off", //This rule enforces consistent line endings independent of operating system, VCS, or editor used across your codebase.
+    "react/require-default-props":"off", //This rule aims to ensure that any non-required prop types of a component has a corresponding defaultProps value.
+    "import/extensions": [
+      "error",
+      "ignorePackages",
+      {
+        "js": "never",
+        "jsx": "never",
+        "ts": "never",
+        "tsx": "never"
```
Comment (bob606) [inline .eslintrc.json:33]: The comments next to the rule disables are supposed to give a profound reason why do disable the rule. Also in general, could you please give some more explanation for your changes.

**Review by bob606 — COMMENTED** [review]
Looks good to me, just the eslint part needs a bit explanation for me 🚀

File: .eslintrc.json (hunk 1 above)
Comment (meshackm) [reply .eslintrc.json:33]: I have added a description for that rule

**Review by bob606 — APPROVED** [review]
good work

**Pull request merged**